Cabal/Distribution/Simple/Build.hs

```diff
@@ -406,7 +408,7 @@ testSuiteLibV09AsLibAndExe pkg_descr
   where
     bi  = testBuildInfo test
     lib = Library {
-            libName = testName test,
```

Comment:
Well, all of the code here is suspicious, really. The choice of libName here doesn't make a difference, it's never used later.